pkg/workloads/console/controller.go

```diff
@@ -180,12 +188,24 @@ func (r *reconciler) createOrUpdate(expected reconcile.ObjWithMeta, kind string,
 		return err
 	}
 
-	r.logger.Log("event", "CreatedOrUpdated", "outcome", outcome, "resource", kind)
+	r.logger.Log("resource", kind, "event", "CreateOrUpdate", "outcome", outcome)
 	return nil
 }
 
 func (r *reconciler) updateStatus(job *batchv1.Job) error {
-	newStatus := calculateStatus(r.console, job)
+	// Fetch the job's pod
+	podList := &corev1.PodList{}
+	pod := &corev1.Pod{}
+	opts := client.
+		InNamespace(r.name.Namespace).
+		MatchingLabels(map[string]string{"job-name": job.ObjectMeta.Name})
+	r.client.List(r.ctx, opts, podList)
```

Comment:
We're missing a check on the error result here

Reply:
Good spot! Will fix.